### PR TITLE
Feature/modify layout n routes

### PR DESCRIPTION
**Pull request by rohmad-st**
This branch contain some changes:
- Modify Layout Top Navbar
- Modify Routes Dashboard
- Add named routes

Now we can call url with name alias
```url('name-alias-routes')```
or set active url by name
with step
1. Add params name in *controllers/file*. Example in *controllers/dashboard.js*
2. From header or similar view can use 
*class=(name === 'name-alias-of-route') ? 'active' : undefined*

For more information about named-route you can check at: 
https://github.com/alubbe/named-routes

### CODE REVIEW

**Comment by haydarai** [comment]
The changes LGTM, only need to resolve the conflict.

File: controllers/dashboard.js [hunk 1]
```diff
@@ -3,37 +3,43 @@
...
+    name: req.route.name,
     title: 'Manage Non-Members'
   });
 };
 
-exports.guestLogs = (req, res) => {
-  res.render('dashboard/guest-logs', {
-    title: 'Guest Logs'
+exports.visitorLogs = (req, res) => {
+  res.render('dashboard/visitor-logs', {
```
Comment (dwiajik) [inline controllers/dashboard.js]: Why is this renamed to visitor-logs? I think this will make confusion because in the title in front end says "Guest Logs".

Comment (rohmad-st) [reply controllers/dashboard.js]: @dwiajik Because that requested from @kuriakinzeng bro, I think also need change title ya?

Comment (dwiajik) [reply controllers/dashboard.js]: Oh okay, just ask @kuriakinzeng about the title then :joy:

**Comment by kuriakinzeng** [comment]
Well done. Just one small tweak: Remove CREATE ACCOUNT. No one should be able to self-register

**Comment by kuriakinzeng** [comment]
Oh I see another PR that removes CREATE ACCOUNT. Ok can merge. @haydarai @rohmad-st

File: controllers/dashboard.js (hunk 1 above)
Comment (kuriakinzeng) [reply controllers/dashboard.js]: Yup let's call it Visitor Logs
